packages/ember-data/lib/system/record-array-manager.js

```diff
@@ -160,6 +171,17 @@ export default Ember.Object.extend({
   },
 
   /**
+    Get the `DS.RecordArray` for a type.
```

Comment:
Add comment that the `RecordArray` returned by this function holds *all* loaded records in the store of the specifc type.